src/cascadia/TerminalSettingsEditor/ActionsViewModel.cpp

```diff
@@ -18,383 +24,1194 @@ using namespace winrt::Windows::UI::Xaml::Data;
+            INITIALIZE_FLAG_LIST_AND_VALUE(SuggestionsSource, Model::SuggestionsSource, L"Actions_SuggestionsSource", L"Content");
+        }
+        else if (_type == L"FindMatchDirection")
+        {
+            INITIALIZE_ENUM_LIST_AND_VALUE(FindMatchDirection, Model::FindMatchDirection, L"Actions_FindMatchDirection", L"Content");
+        }
+        else if (_type == L"Model::DesktopBehavior")
+        {
+            INITIALIZE_ENUM_LIST_AND_VALUE(DesktopBehavior, Model::DesktopBehavior, L"Actions_DesktopBehavior", L"Content");
+        }
+        else if (_type == L"Model::MonitorBehavior")
+        {
+            INITIALIZE_ENUM_LIST_AND_VALUE(MonitorBehavior, Model::MonitorBehavior, L"Actions_MonitorBehavior", L"Content");
+        }
+        else if (_type == L"winrt::Microsoft::Terminal::Control::ClearBufferType")
+        {
+            INITIALIZE_ENUM_LIST_AND_VALUE(ClearBufferType, Control::ClearBufferType, L"Actions_ClearBufferType", L"Content");
+        }
+        else if (_type == L"SelectOutputDirection")
+        {
+            INITIALIZE_ENUM_LIST_AND_VALUE(SelectOutputDirection, Model::SelectOutputDirection, L"Actions_SelectOutputDirection", L"Content");
+        }
+        else if (_type == L"Model::SplitDirection")
+        {
+            INITIALIZE_ENUM_LIST_AND_VALUE(SplitDirection, Model::SplitDirection, L"Actions_SplitDirection", L"Content");
+        }
+        else if (_type == L"SplitType")
+        {
+            INITIALIZE_ENUM_LIST_AND_VALUE(SplitType, Model::SplitType, L"Actions_SplitType", L"Content");
+        }
+        else if (_type == L"Windows::Foundation::IReference<TabSwitcherMode>")
+        {
+            INITIALIZE_NULLABLE_ENUM_LIST_AND_VALUE(TabSwitcherMode, Model::TabSwitcherMode, L"Actions_TabSwitcherMode", L"Content");
+        }
+        else if (_type == L"Windows::Foundation::IReference<Control::CopyFormat>")
+        {
+            INITIALIZE_NULLABLE_FLAG_LIST_AND_VALUE(CopyFormat, Control::CopyFormat, L"Actions_CopyFormat", L"Content");
+        }
+        else if (_type == L"Windows::Foundation::IReference<Microsoft::Terminal::Core::Color>" ||
+                 _type == L"Windows::Foundation::IReference<Windows::UI::Color>")
+        {
+            ColorSchemeRequested.raise(*this, *this);
+        }
+        else if (_tag == Model::ArgTag::ColorScheme)
+        {
+            // special case of string, emit an event letting the actionsVM know we need the list of color scheme names
+            ColorSchemeNamesRequested.raise(*this, *this);
+
+            // even though the arg type is technically a string, we want an enum list for color schemes specifically
+            std::vector<winrt::Microsoft::Terminal::Settings::Editor::EnumEntry> namesList;
+            const auto currentSchemeName = unbox_value<winrt::hstring>(_Value);
+            auto nullEntry = winrt::make<winrt::Microsoft::Terminal::Settings::Editor::implementation::EnumEntry>(RS_(L"Actions_NullEnumValue"), nullptr);

```

Comment:
This may be where you can add a custom value for the `null` enum value. Maybe hardcode a `Name` check so that we do a different one depending on the setting?